### DISCUSSION TITLE
Constant notifications for LinuxServer commits

### DISCUSSION THREAD

**Discussion by kingp0dd** [crazy-max/diun#893] [Q&A]
I have multiple images from LinuxServer and Diun picks up their commits even if there's no version/release change.

Is there a way to only be notified of Releases instead of commits?

**Comment by crazy-max** [comment]
Not sure what provider you're using but with [Docker provider](https://crazymax.dev/diun/providers/docker/) you can use [`diun.include_tags` or `diun.exclude_tags` labels](https://crazymax.dev/diun/providers/docker/#docker-labels).

**Comment by kingp0dd** [comment]
The versioning of LinuxServer's images is like this:
```
1.7.4.3770-ls28
1.7.4.3769-ls27
1.7.4.3769-ls26
1.7.4.3769-ls25
```
`ls27` to `ls25` are minor releases (bumping of linux utilities). Is there a way to just get notified when there's a non-minor release (in this ex is `ls28`)?

**Reply by kingp0dd** [reply]
Can diun.include_tags work like this:

`/^(.*)-ls*$/`

so Diun will only compare if there's a change in:
```
1.7.4.3770
1.7.4.3769
1.7.4.3769
1.7.4.3769
```

**Reply by mjlynden** [reply]
if it helps, I've successfully used the below regex to only include image tags like `4.0.16.2944-ls301` for v4 of the image.

The `\r?` makes the carriage return optional, which handles both Windows (\r\n) and Unix (\n) line endings.

`^4\.\d+\.\d+\.\d+-ls\d+\r?$`
